docs/releases/changelog-dev.md

```diff
@@ -2,8 +2,82 @@
 
 ### New features since last release
 
+- Implemented Crosstalk automatic implementation through the experiment class. The crosstalk can be added through the `Calibration` file or by creating a `CrosstalkMatrix`. The crosstalk implementation inside the `Experiment` class while not using `Calibration` should be similar to this:
+
+```
+experiment = ql.Experiment(label="liveplot_test")
+
+flux_x = experiment.variable("flux_x", ql.Domain.Flux)
+flux_z = experiment.variable("flux_z", ql.Domain.Flux)
+
+experiment.crosstalk(crosstalk=crosstalk_matrix)  # to see the values to be applied on the sample
+with experiment.for_loop(variable=flux_x, start=0, stop=0.4, step=0.01):
+    with experiment.for_loop(variable=flux_z, start=0, stop=0.4, step=0.01):
+        experiment.set_parameter(alias="flux_x1", parameter=ql.Parameter.FLUX, value=flux_x)
+        experiment.set_parameter(alias="flux_z1", parameter=ql.Parameter.FLUX, value=flux_z)
+        experiment.execute_qprogram(qp)
+```
+
+With `Calibration`, `experiment.crosstalk` requires the calibration file:
+
+```
+experiment = ql.Experiment(label="liveplot_test")
+
+flux_x = experiment.variable("flux_x", ql.Domain.Flux)
+flux_z = experiment.variable("flux_z", ql.Domain.Flux)
+
+experiment.crosstalk(calibration=calibration)
+with experiment.for_loop(variable=flux_x, start=0, stop=0.4, step=0.01):
+    with experiment.for_loop(variable=flux_z, start=0, stop=0.4, step=0.01):
+        experiment.set_parameter(alias="flux_x1", parameter=ql.Parameter.FLUX, value=flux_x)
+        experiment.set_parameter(alias="flux_z1", parameter=ql.Parameter.FLUX, value=flux_z)
+        experiment.execute_qprogram(qp)
+```
+
+Note that not giving a crosstalk matrix implies working with the identity. Warnings rise while creating the experiment to inform the user of this.
+[#899](https://github.com/qilimanjaro-tech/qililab/pull/899)
```

Comment:
Take a look in the code example and make sure it's updated to the latest changes, e.g. `set_crosstalk` instead of `add_crosstalk` and so on.